src/app/main/component/ubs/components/ubs-order-details/ubs-order-details.component.ts

```diff
@@ -72,6 +90,10 @@ export class UBSOrderDetailsComponent implements OnInit, OnDestroy {
     });
   }
 
+  getFormValues(): boolean {
+    return this.showTotal > 0 ? true : false;
```

Comment:
lets simplify this: return this.showTotal > 0;

Reply:
this line has been shortened with this commit https://github.com/ita-social-projects/GreenCityClient/pull/896/commits/43890ba93b0ec853b4de3ab164c0da4d997db727